content/blog/2023-06-19-develop-your-cloudflare-workers-aws-apps-locally-with-localstack-miniflare/index.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Develop your Cloudflare Workers + AWS apps locally with LocalStack & Miniflare
+description: Develop your Cloudflare Workers + AWS apps locally with LocalStack & Miniflare
+lead: Develop your Cloudflare Workers + AWS apps locally with LocalStack & Miniflare
+date: 2023-06-19T3:27:34+05:30
+lastmod: 2023-06-19T3:27:34+05:30
+images: []
+contributors: ["Harsh Mishra", "Waldemar Hummer", "Brendan Coll", "Adam Murray"]
```

Comment:
nit: We could slightly adjust the list of authors - LS first, then CF, and alphabetically by first name within each group.
```suggestion
contributors: ["Harsh Mishra", "Waldemar Hummer", "Adam Murray", "Brendan Coll"]
```

When we publish the blog on CF side, I'd expect that we'll adjust the order (CF first, then LS)